UnityProject/Assets/Scripts/Core/Transform/CustomNetTransform.Move.cs

```diff
@@ -366,7 +366,7 @@ public void Throw(ThrowInfo info)
 		//add player momentum
 		float playerMomentum = 0f;
 		//If throwing nearby, do so at 1/2 speed (looks clunky otherwise)
-		float speedMultiplier = Mathf.Clamp(correctedInfo.WorldTrajectory.magnitude / (throwRange <= 0 ? 1 : throwRange), 0.6f, 1f);
+		float speedMultiplier = Mathf.Clamp(correctedInfo.WorldTrajectory.magnitude / (throwRange <= 0 ? 1 : throwRange), 0.2f, 1f);
```

Comment:
how is this change with normal throwing?

Reply:
humm, only difference I noticed is that stuff like ID cards throw really slowly if you throw them close, but I just imagine that as them sliding along the floor for ages

Reply:
if I don't have this it snaps beyond the target position too fast, and instantly stops, make it look like it just appears there  , if the target position is close to the origin of the body

Reply:
Should we make throw force a parameter?

Reply:
A bit odd that we don't already

> 
> 
> Should we make throw force a parameter?

A bit odd that we don't already.

Reply:
I think one of the issues is that it expects a target position, to slow down in such not too sure though the code is a bit hard to read